core/src/main/scala/org/allenai/common/Version.scala

```diff
@@ -81,19 +83,18 @@ object Version {
     val pkg = "/" + org + "/" + name.replaceAll("-", "")
     val artifactConfUrl = this.getClass.getResource(pkg + "/artifact.conf")
     val gitConfUrl = this.getClass.getResource(pkg + "/git.conf")
-
+    val cacheKeyConfUrl = this.getClass.getResource(pkg + "/cacheKey.conf")
```

Comment:
It shouldn't - that should just make it remove the old staging dir before starting the main stage.